src/backends/backend/examples/backend_utils.cc

```diff
@@ -135,6 +151,126 @@ ShapeToString(const std::vector<int64_t>& shape)
   return ShapeToString(shape.data(), shape.size());
 }
 
+std::string
+DataTypeToString(TRITONSERVER_DataType dtype)
```

Comment:
Instead of this should use TRITONSERVER_DataTypeString which is now available in TRITONSERVER API.